src/dbnode/integration/options.go

```diff
@@ -91,6 +92,9 @@ var (
 	DefaultIntegrationTestRetentionOpts = retention.NewOptions().SetRetentionPeriod(6 * time.Hour)
 )
 
+// StorageOptsFunc is a storageOptions modifier for tests
+type StorageOptsFunc func(opts storage.Options) storage.Options
+
```

Comment:
There's already support for passing `storage.Options` mutators in `NewTestSetup()`. I believe a similar varargs parameter could be added to `NewDefaultBootstrappableTestSetups()`, which would then be passed to `NewTestSetup()`

https://github.com/m3db/m3/blob/31a21e3792119e3d7588c882119b6115b9486060/src/dbnode/integration/setup.go#L189-L197

https://github.com/m3db/m3/blob/31a21e3792119e3d7588c882119b6115b9486060/src/dbnode/integration/integration.go#L214

Reply:
Thanks. I was wondering that I saw it somewhere.